pybrowscap/test/loader/test_downloader.py

```diff
@@ -1,61 +0,0 @@
-import os
```

Comment:
Why removing the test here ?

Reply:
sorry, I delete this test, but I don't think it is necessary.